data/json/items/gun/44.json

```diff
@@ -1,5 +1,34 @@
 [
   {
+    "id": "cane_rifle",
+    "type": "GUN",
+    "name": "walking cane",
```

Comment:
Briefcase SMG, which is significantly less gun-like, clearly states that it is a briefcase SMG.

The hidden name gimmick bullshit gets boring fast.